tests/core/engines/alpha/test_guideline_proposer.py

```diff
@@ -223,7 +225,11 @@ def propose_guidelines(
     terms: Sequence[Term] = [],
     staged_events: Sequence[EmittedEvent] = [],
 ) -> Sequence[GuidelineProposition]:
-    guideline_proposer = GuidelineProposer(context.logger, context.schematic_generator)
+    guideline_proposer = GuidelineProposer(
+        context.logger,
+        context.schematic_generator,
+        context.container[ShotCollection[GuidelinePropositionShot]],
```

Comment:
You don't need to provide it to the guideline proposer like this -- it's already accessible to it as part of its module.